task/common/resource.go

```diff
@@ -0,0 +1,28 @@
+package common
+
+import (
+	"context"
+)
+
+// BlobContainer defines the interface implemented by blob container resources
+// across cloud providers.
+type BlobContainer interface {
+	Resource
+	// Identifier returns the identifier of the container.
+	Identifier() string
+}
```

Comment:
Where are we using this interface? 🤔

Reply:
Ah, good catch - this is a left-over from a previous implementation attempt.

Reply:
Resolved with https://github.com/iterative/terraform-provider-iterative/pull/651/commits/07e4116a3ff87fdb6ba69aeb224b723fa1e5840f